utils/analysis/ab_analysis.py

```diff
@@ -30,11 +30,10 @@
 
 # Set of metrics we are interested in, along with a function to extract that
 # metric from a WPT result row.
+# TODO(csharrison): Add ParseTime when we have valid start/end parse times.
 _METRICS = {
     'SpeedIndex': lambda x: sanitizeInt(x['SpeedIndex']),
     'TTFP': lambda x: sanitizeInt(x['render']),
-    'ParseTime': lambda x: sanitizeIntDiff(
-        x['chromeUserTiming.domInteractive'], x['chromeUserTiming.domLoading']),
```

Comment:
I just pushed the domLoading stand-alone metric so "domInteractive" and "domLoading" should both be available from the metrics directly without the need for user timing.